base/applications/cmdutils/at/lang/it-IT.rc

```diff
@@ -0,0 +1,63 @@
+/*
+ * PROJECT:     ReactOS AT Utility
+ * LICENSE:     See COPYING in the top level directory
+ * PURPOSE:     Italian resource file
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+LANGUAGE LANG_ITALIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    IDS_USAGE "Il comando AT permette di stabilire altri comandi e programmi di eseguire\n\
+in un computer in un certo tempo. Lo servizio "Schedule" deve essere in esecuzione per\n\
+poter usare il comando AT.\n\n\
+AT [\\\\nomecomputer] [ [id] [/DELETE] | /DELETE [/YES]]\n\
+AT [\\\\nomecomputer] tempo [/INTERACTIVE]\n\
+    [ /EVERY:data[,...] | /NEXT:data[,...]] ""comando""\n\n\
+\\\\nomecomputer     Specifica un computer remoto. I comandi sono stabiliti in un \n\
+                   computer locale se questo parametro è omesso.\n\
+id                 È un numero d'identificazione assegnato in un comando\n\
+                   stabilito.\n\
+/DELETE            Cancella un comando stabilito. Se l'id viene omesso, tutti\n\
+                   i comandi stabiliti nel computer saranno cancellati.\n\
+/YES               Usato con l'opzione "DELETE", annullerà tutti i requisiti\n\
```

Comment:
Wrong translation of *Scheduled* so *"[...] annullerà tutti i compiti programmati nel computer."*